### PR TITLE
Added ProgressMeter to transfer_to_memory function (Issue #1028 )

### LINKED ISSUE #1028: transfer_to_memory should have a ProgressMeter
### Expected behaviour

This should report progress of loading trajectory to memory. This is nice feedback for large trajectories (1GB xtc file for example)

``` python
u.transfer_to_memory(quiet=False)
```
### Actual behaviour

The kwarg isn't accepted.
### How to fix

Add the `ProgressMeter` from `MDAnalysis.lib.log` to the loop reading the coordinates.
### Currently version of MDAnalysis:

(run `python -c "import MDAnalysis as mda; print(mda.__version__)"`)
0.16.0dev0

### PR DESCRIPTION
**Pull request by shanmbic**
Fixes #1028

Changes made in this Pull Request:

Added ProgressMeter to the "transfer_to_memory" function with a 'quiet' parameter.

### CODE REVIEW

File: package/MDAnalysis/core/AtomGroup.py [hunk 1]
```diff
@@ -5046,21 +5047,27 @@ def _matches_unpickling(self, anchor_name, n_atoms, fname, trajname):
...
+                 set to True. Default value is quiet. 
 
         .. versionadded:: 0.15.0
         """
 
         from ..coordinates.memory import MemoryReader
 
         if not isinstance(self.trajectory, MemoryReader):
+            
+            percentage = ProgressMeter(self.trajectory.n_frames,
```
Comment (kain88-de) [inline package/MDAnalysis/core/AtomGroup.py]: we normally use `pm` as variable name for a ProgressMeter

**Comment by kain88-de** [comment]
@shanmbic can you also update the CHANGELOG and add your self to the authors list. Thanks.

**Comment by kain88-de** [comment]
@MDAnalysis/coredevs I'll merge this later by hand into the `issue-363` branch and only have the fix there. This will ensure that this problem stays fixed after we merge the new topology system.

**Comment by shanmbic** [comment]
I have updated the CHANGELOG and changed the ProgressMeter Variable . @kain88-de

**Comment by kain88-de** [comment]
Please also add yourself to the authors file thanks.

https://github.com/MDAnalysis/mdanalysis/blob/develop/package/AUTHORS

**Comment by kain88-de** [comment]
merged into `issue-363` branch. Thanks

**Pull request closed without merging**